src/redux/themeSlice.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { localStorageKeys } from '../auxFiles/localStorageKeys';
+
+const {
+  themesKeys: {
+    OSTheme,
+    theme,
+  },
+} = localStorageKeys;
+
+const currentOSTheme = localStorage.getItem(OSTheme) || '';
+const themeNameFromLocalStorage = localStorage.getItem(theme) || '';
+
+type State = {
+  name: string,
+};
+
+const initialState: State = { name: themeNameFromLocalStorage || currentOSTheme };
+
+const themeSlice = createSlice({
+  name: 'selectedTheme',
+  initialState,
```

Comment:
I suggest it's ok to store initialState object right in the createSlice declaration. It's easier to read.